README.md

```diff
@@ -1,16 +1,33 @@
 # mbed-client-cli
 
-This is the Command Line Library for a CLI application. This library provides methods for:
+This is the Command Line Library for a CLI application. It uses only ansi C features so it is portable and works in mbed-os-3, mbed-os-5, linux and windows.
+
+## Features
+
+Library provides features such:
 
 * Adding commands to the interpreter.
 * Deleting commands from the interpreter.
 * Executing commands.
 * Adding command aliases to the interpreter.
 * Searching command arguments.
+* implements several VT100/VT220 features, e.g.
+  * move cursor left/right (or skipping word by pressing alt+up/right)
+  * delete characters
+  * CTRL+W to remove previous word
+  * browse command history by pressing up/down
+* implements basic commands, e.g.
+  * echo
+  * help
+  * (un)set
+  * alias
+  * history
+  * true/false
```

Comment:
same as true/false, wasn't in the mbed-os-example.bin help message

Reply:
I did not add any title/mans for this commands to save some memory.. maybe there could be some short text.. but help should still print them.. hmm have to check it.

Reply:
Yep it should be printed, see https://github.com/ARMmbed/mbed-client-cli/pull/76#discussion_r207989630